flutter/multi_user_sub.dart

```diff
@@ -0,0 +1,3 @@
+channel.subscribe(name: 'update').listen((message) {
```

Comment:
Why do you put the `.method` on separate lines in the first example, yet not here?

Reply:
It was because I used an arrow function in the first one. However, it looks better when it is put in multiple lines as the method bodies are small. Necessary changes here: bce771d587ed3bac4425f14f209be479907c0dbb